Marlin/src/pins/ramps/pins_RAMPS.h

```diff
@@ -251,9 +251,12 @@
   #define FILWIDTH_PIN                         5  // Analog Input on AUX2
 #endif
 
-// RAMPS 1.4 DIO 4 on the servos connector
 #ifndef FIL_RUNOUT_PIN
-  #define FIL_RUNOUT_PIN                       4
+  #if ENABLED(CR20_STOCKDISPLAY)
+    #define FIL_RUNOUT_PIN                     2
```

Comment:
I doubt this will work for every RAMPS board. Does the CR-20 have a board that's different from other Creality v2.x boards?

Reply:
It uses Creality's V2.2 board, every schematic I've looked at for the board shows the same port for the filament runout sensor. You can leave this out, the contrast adjustment is the more important change.